src/string.js

```diff
@@ -20,7 +21,12 @@ export default function StringSchema() {
   this.withMutation(() => {
     this.transform(function (value) {
       if (this.isType(value)) return value;
-      return value != null && value.toString ? value.toString() : value;
+      if (Array.isArray(value)) return value;
+
+      const strValue =
+        value != null && value.toString ? value.toString() : value;
+      if (strValue === objTag) return value;
```

Comment:
is `typeof` no good here? i guess you'd have to do `instanceof String` as well in case there's a `new String('foo')`?